lib/auth.js

```diff
@@ -140,11 +140,12 @@ const loginPasswordTests = {
 };
 
 class AuthenticationStrength {
-  constructor(
-    valid, // Boolean
-    hints, // Object, { required: Array, optional: Array }
-    compliance // Number, ratio of passed optional tests to all optional tests
-  ) {
+  // AuthenticationStrength constructor
+  //   valid - <boolean>
```

Comment:
Shouldn't boolean be uppercased too? (refs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Boolean)

Reply:
I thought that types should be the same as in nodejs docs. Like [here](https://nodejs.org/api/path.html#path_path_isabsolute_path).

Reply:
@lundibundi, no, `boolean` is a scalar type.

Reply:
Oh, my bad.